runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/stableinput/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Classes for buffering elements for achieving @RequiresStableInput. */
+package org.apache.beam.runners.flink.translation.wrappers.streaming.stableinput;
```

Comment:
WDYT about NonNull by default here (and elsewhere)? We've caught - and missed - critical NPE bugs using that.

Reply:
Ah, didn't know that annotating `@ParametersAreNonnullByDefault` here would enforce non null parameters. Will try that out.

Reply:
Here is what I did a while ago to most modules: https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/package-info.java

But actually this misses some things, so I made this but have not migrated to it: https://github.com/apache/beam/blob/master/sdks/java/build-tools/src/main/java/org/apache/beam/buildtools/NonnullByDefault.java

Reply:
Cool. I've added `NonnullByDefault`, didn't have to fix anything. We can expand it to the entire Flink Runner.